packages/material-ui-styles/src/styled/styled.js

```diff
@@ -82,6 +84,19 @@ export default function styled(Component) {
       const classes = useStyles(props);
       const className = clsx(classes.root, classNameProp);
 
+      if (process.env.NODE_ENV !== 'production') {
+        if (!warnedOnce && props.css) {
```

Comment:
Use `.propTypes` instead. That way we get a component stack during SSR and a better de-duplication strategy.

Reply:
Yes, please! I still can't figure out where this warning in the documentation is coming from:
<img width="515" alt="Capture d’écran 2020-11-12 à 10 23 52" src="https://user-images.githubusercontent.com/3165635/98921155-43d92080-24d1-11eb-8e8a-fb3a3c7cbe63.png">

Reply:
Done

Reply:
> Yes, please! I still can't figure out where this warning in the documentation is coming from:

Should be fixed with https://github.com/mui-org/material-ui/pull/23285 but I wanted to wait for #23294 since we might get way too many warnings otherwise.